src/st/dre/22848.c

```diff
@@ -9,8 +9,61 @@ long RotTransPers3(SVECTOR* v0, SVECTOR* v1, SVECTOR* v2, long* sxy0,
                    long* sxy1, long* sxy2, long* p, long* flag);
 long NormalClip(long sxy0, long sxy1, long sxy2);
 
-Primitive* func_801A2848(Primitive* prim, u8* model, u16* arg2);
-INCLUDE_ASM("st/dre/nonmatchings/22848", func_801A2848);
+Primitive* func_801A2848(Primitive* prim, u8* arg1, u16* arg2) {
+    s32 pad0;
+    s32 pad1;
+    s32 pad2;
+    s32 pad3;
+    s32 pad4;
+
+    pad0 = *SPAD(arg1[0]);
```

Comment:
Wanna use `SP` instead?

Reply:
What if SPAD internally uses SP? Reason is the scratch pad offset on PSP is different and on PC I got far too many unexpected crashes not knowing a hardcoded address was used for the scratchpad. Using SP, even if SPAD is wrapping it, will at least a compilation error

Reply:
Sure, that sounds like a nice idea. Will implement.